js-sample/src/js/app.js

```diff
@@ -39,7 +43,10 @@ var testStreamingCapability = function(subscriber, callback) {
     publisher.publishVideo(false);
 
     performQualityTest({subscriber: subscriber, timeout: 5000}, function(error, results) {
-      if (results.audio.packetLossRatioPerSecond < 0.05) {
+      var audioSupported = results.audio.bitsPerSecond > 25000
```

Comment:
In one scenario we pass audio if it's packet loss is under `0.05`, but in this case, we also check that it's `bitsPerSecond` is greater than `25,000`? Is this intentional?